qt/bundle.h

```diff
@@ -40,6 +40,11 @@ class APPSTREAMQT_EXPORT Bundle {
         Bundle& operator=(const Bundle& bundle);
         bool operator==(const Bundle& r) const;
 
+        /**
+         * \returns the internally stored AsBundle
+         */
+        _AsBundle *asBundle() const;
```

Comment:
Then we should make the symbols internal and hide them from the public API... The tricky bit is that they're class members...
(I will take a look on whether it's possible to even have them as internal members, probably the only way to do that is to make them private and use `friend` classes for access. - in any case, just leave this as it is for now :-) )